### PR TITLE
PR with more strict travis setup checking build on every push

### PR DESCRIPTION
**Pull request by aorumbayev**
Every push on every branch triggers build stage to check whether code compiles on both backend and frontend

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +12,42 @@ stages:
...
+      cache:
+        directories:
+          - $HOME/.gradle/caches/
+          - $HOME/.gradle/wrapper/
+      services:
+        - postgresql
+      before_script:
+        - cd src/backend
+        - psql -c "CREATE DATABASE lpa;" -U postgres
+        - psql -c "CREATE USER lpa WITH PASSWORD 'example';" -U postgres
```
Comment (eghuro) [inline .travis.yml]: I guess it's not needed here. However, if it is, then also add `- ./gradlew flywayClean flywayMigrate`

Comment (eghuro) [inline .travis.yml]: I guess it's no longer needed here for assembling only.

Comment (aorumbayev) [reply .travis.yml]: @eghuro what do u mean ?

Comment (eghuro) [reply .travis.yml]: See this change: https://github.com/linkedpipes/applications/compare/ad0d06cb457ee139fee44728f001a251d0e94414...38f8086a51ba85d1eb9b74f7d9edcedb20b5bc26

**Pull request merged**